MiniApp/Classes/MiniApp.swift

```diff
@@ -52,14 +52,23 @@ public class MiniApp: NSObject {
         return realMiniApp.createMiniApp(appId: appId, completionHandler: completionHandler, messageInterface: messageInterface)
     }
 
+    /// Create a Mini App for the given mini url.
```

Comment:
For now the ability to load from a URL is supposed to be only used by our Demo App, so we don't really want to support clients using it in their own App yet.

Is there any way to hide a function from the generated documentation? Or restrict it's usage to the example app? For example, on Android we can hide it from the docs using `@suppress` or use `@RestrictTo` to limit it's usage to a specific package. 

If not, probably it's okay. We can just leave out this new function from the user guide so we aren't advertising that it exists.

Reply:
Unfortunately we don't have this in Swift. The most elegant thing that comes to my mind about this would be to create a subspec dedicated to this feature, `MiniApp/Sample` (https://guides.cocoapods.org/syntax/podspec.html#subspec). I can arrange this if you want

Reply:
Subspec sounds like a good idea. It's either this or removal of documentation for that new method. Which one do you gyus think would be the best solution?

Reply:
Hmm, I think it's probably okay to leave it as is. Eventually we might want to open up this method for general use anyways.

Maybe update the doc for this method to say it's for previewing a mini app? i.e. `This should only be used for previewing a mini app from a local server.`